# Data acquisition time and frame id
std_msgs/Header header

# Object CAN messages
conti_ars408_side2/radar_object_info[] object_info